src/transformers/models/clipseg/modeling_clipseg.py

```diff
@@ -833,6 +833,7 @@ def forward(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
+        interpolate_pos_encoding: bool = False,
```

Comment:
In this case, I would say we can remove the `copied from` here. The forward pass is very simple / standard and so there isn't anything critical being copied and the embeddings will always be interpolated for this model when needed

Reply:
Originally I do not want to change clipseg since it implement the same function itself and even more aggressive, but `make fix-copies` add this for me.
And without it, the pr cannot pass consistency check, could you kindly provide some suggestions?

Reply:
You can remove the copied from header on L829. The forward pass logic is very generic so removing it here isn't an issue.